app/scripts/modules/core/src/pipeline/config/stages/overrideTimeout/overrideTimeout.directive.js

```diff
@@ -38,19 +38,28 @@ module.exports = angular
           stageConfig = Registry.pipeline.getStageConfig(stage),
           stageDefaults = stageConfig ? stageConfig.defaultTimeoutMs : null;
 
+        // vm.overrideTimeout being false is only a transition state to indicate the override got unchecked
```

Comment:
super minor, but wanted to point out that you could better indicate the relationship between these vars like: ```
         // Capturing this so that we can restore it after $scope.vm is clobbered
        const originalOverrideTimeout = $scope.vm && $scope.vm.overrideTimeout;

        // vm.overrideTimeout being false is only a transition state to indicate the override got unchecked
        const shouldRemoveOverride = originalOverrideTimeout === false;
```

Reply:
Great idea. Thanks!